applications/IgaApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -20,14 +20,204 @@
 #include "processes/process.h"
 #include "custom_python/add_custom_utilities_to_python.h"
 
-#include "spaces/ublas_space.h"
-#include "linear_solvers/linear_solver.h"
+#include "iga_application_variables.h"
+#include "custom_utilities/node_curve_geometry_3d.h"
+#include "custom_utilities/node_surface_geometry_3d.h"
 
 namespace Kratos {
 namespace Python {
 
-void  AddCustomUtilitiesToPython(pybind11::module& m)
+void AddCustomUtilitiesToPython(pybind11::module& m)
 {
+    namespace py = pybind11;
+    using namespace pybind11::literals;
+
+    using Array3D = Kratos::array_1d<double, 3>;
```

Comment:
I thing better to use a typedef that a using (using are evil)

Reply:
[Is there any difference?](https://stackoverflow.com/questions/10747810/what-is-the-difference-between-typedef-and-using-in-c11)

Reply:
Ok you won this argument